scanpipe/pipes/d2d.py

```diff
@@ -38,6 +41,37 @@
 IGNORE_EXTENSIONS = ()
 IGNORE_PATHS = ("gradleTest/",)
 
+PROSPECTIVE_JAVASCRIPT_MATCH = {
+    ".scss": [
+        {
```

Comment:
Could you add some comments to describe what this is? Or add a field in the dictionary with some help text?